docs/Hubspot-batchsource.md

```diff
@@ -12,7 +12,7 @@ Properties
 
 **API Key:** OAuth2 API Key.
 
-**Object(s) to pull:** Names of Object(s) to pull from Hubspot. Select from Contact Lists, Contacts, Email Events, 
+**Object to pull:** Name of Object to pull from Hubspot. Select from Contact Lists, Contacts, Email Events, 
```

Comment:
Seems like the description is not matching with the doc
https://cdap.atlassian.net/wiki/spaces/DOCS/pages/718798896/Hubspot+Batch+Source